### ISSUE TITLE
Animation for tooltips that have `data-balloon-visible`

### ISSUE THREAD

**Issue by ghost** [kazzkiq/balloon.css#136]
In one of my recents projects I used `data-balloon-visible` attribute through javascript for alert the user if there is any kinda of error on the form's inputs.
I noticed that when the tooltip appear, there isn't any animation at all.

## Step to reproduce:
Simply try to use a balloon tooltip with an `data-baloon-visible` setted through javascript.

### Expected Behavior
The tooltip came up with an animation 

### Real Behavior
There isn't any animation at all

**Comment by moeenio** [comment]
I can't reproduce, are you sure you're not using the `data-balloon-blunt`attribute?

**Comment by ghost** [comment]
Ok,
I did some research and testing, link [here](https://jsfiddle.net/cdmsk5e2/).
We have the problem only when we set the `aria-label` through js, otherwise all work fine.

**Comment by moeenio** [comment]
If you wanna contribute, I think you should look [here](https://github.com/kazzkiq/balloon.css/blob/master/src/balloon.scss#L66).

**Comment by ghost** [comment]
here were the tooltip blow up? 
https://github.com/kazzkiq/balloon.css/blob/f30698391242ba4ca8848de1c924903664e55212/src/balloon.scss#L113

Also, if you  are interested I  did some other tests (link [here](https://jsfiddle.net/pfgkcsue/)), if you set the `data-balloon-visible` after `aria-label` all work well. Interesting behavior.

**Comment by moeenio** [comment]
Yes, it's this part of the code that makes the tooltip show up. This behaviour is justified because it's the addition of the `data-balloon-visible` attribute that triggers the animation, that's the part of the code I pointed at.
